app/assets/javascripts/components/articles/article.jsx

```diff
@@ -56,7 +60,7 @@ const Article = createReactClass({
           </div>
         </td>
         <td className="desktop-only-tc">{this.props.article.character_sum}</td>
-        <td className="desktop-only-tc">{this.props.article.references_count}</td>
+        <td className="desktop-only-tc">{refCount}</td>
```

Comment:
This works, but since `0` is falsy in javascript, you could take a shortcut here and do `this.props.article.references_count || ''`.